seml/start.py

```diff
@@ -134,8 +132,14 @@ def start_slurm_job(collection, exp_array, unobserved=False, post_mortem=False,
 
     # Construct Slurm script
     template = pkg_resources.resource_string(__name__, "slurm_template.sh").decode("utf-8")
+    if 'project_root_dir' in exp_array[0][0]['seml']:
+        project_root = exp_array[0][0]['seml']['project_root_dir']
+    else:
+        project_root = exp_array[0][0]['seml']['config_dir']
```

Comment:
Again, the name `project_root` is not a good fit if you use it like this. We could call it `cwd` or `working_dir` instead.